plugins/WORKSPACE

```diff
@@ -69,37 +63,70 @@ http_archive(
     strip_prefix = "quiche-1397c94d55af0bfc142ac7dda923cf2726857755",
     urls = ["https://github.com/google/quiche/archive/1397c94d55af0bfc142ac7dda923cf2726857755.tar.gz"],
 )
-# Quiche dependency.
+
+# For testing, GoogleURL is a Quiche dependency.
 http_archive(
     name = "com_google_googleurl",  # 2022-11-03
     sha256 = "59f14d4fb373083b9dc8d389f16bbb817b5f936d1d436aa67e16eb6936028a51",
     urls = ["https://storage.googleapis.com/quiche-envoy-integration/dd4080fec0b443296c0ed0036e1e776df8813aa7.tar.gz"],
 )
 
+# --- PLUGIN DEPENDENCIES ---
+
+PROXY_WASM_HOST_COMMIT = "04dfb94747e6462b65f7ed694e2d9c78ab207e11"  # 2024-04-29
```

Comment:
bikeshed (optional): I think you may as well name the vars PROXY_WASM_CPP_HOST_* and PROXY_WASM_CPP_SDK_*, just to stay consistent with their well-known names, as well as other symbols already used in this file (e.g. `proxy_wasm_cpp_host_dependencies()`, etc.)

Reply:
Yup, done.